src/main/java/sirius/biz/importer/format/ImportDictionary.java

```diff
@@ -82,20 +84,28 @@ public ImportDictionary addField(FieldDefinition field) {
 
         this.fields.put(field.getName(), field);
 
-        addCheckedAlias(field, field.getLabel());
-        field.getAliases().forEach(alias -> addCheckedAlias(field, alias));
+        Set<String> aliasDuplicatesToRemove = new HashSet<>();
+
+        addCheckedAlias(field, field.getLabel(), aliasDuplicatesToRemove::add);
+        field.getAliases().forEach(alias -> addCheckedAlias(field, alias, aliasDuplicatesToRemove::add));
+
+        aliasDuplicatesToRemove.forEach(field::removeAlias);
 
         return this;
     }
 
-    private void addCheckedAlias(FieldDefinition field, String alias) {
+    private void addCheckedAlias(FieldDefinition field, String alias, Consumer<String> duplicateConsumer) {
         String previousField = aliases.putIfAbsent(normalize(alias), field.getName());
         if (Strings.isFilled(previousField) && !Strings.areEqual(field.getName(), previousField)) {
-            throw new IllegalArgumentException(Strings.apply(
-                    "Cannot add alias '%s' for field '%s', as this alias already points to '%s'!",
-                    alias,
-                    field.getName(),
-                    previousField));
+            duplicateConsumer.accept(alias);
+            Exceptions.handle()
+                      .to(Log.BACKGROUND)
+                      .withSystemErrorMessage(
+                              "Cannot add alias '%s' for field '%s', as this alias already points to '%s'!",
```

Comment:
Message does not fit behaviour. The alias was added, but the previous was removed